### PR TITLE
Switch from BeautifulSoup to lxml

### PR DESCRIPTION
**Pull request by seanwiseman**
As mentioned by @de-code , this change speeds up the parsing considerably.

Once this is in, I can get the `person` consumers in as they use `lxml` natively.

### CODE REVIEW

File: csv_generator/consumers/__init__.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def feed_consumers(file_list: List[str], output_dir: str, create_date: str,
 
     for data_file in file_list:
         LOGGER.debug('consuming: ', data_file)
-        soup = make_soup(os.path.join(zip_dir, data_file))
+        soup = etree.parse(os.path.join(zip_dir, data_file))
```
Comment (de-code) [inline csv_generator/consumers/__init__.py]: very minor probably want to rename `soup` here too

File: csv_generator/consumers/consumer.py [hunk 2]
```diff
@@ -23,9 +21,9 @@ def _delete_old_output_file(self):
             os.remove(self.output_file)
 
     @staticmethod
-    def get_contents(soup: BeautifulSoup, target_ele: str) -> str:
+    def get_contents(ele: 'lxml.etree.ElementTree', target_ele: str) -> str:
```
Comment (de-code) [inline csv_generator/consumers/consumer.py]: very minor might prefer longer name but it's clear

**Comment by de-code** [comment]
Great, thanks for doing it. Resolves #33

**Comment by seanwiseman** [comment]
No problem. It had to happen. I did not actually realize how slow BeautifulSoup could be.

**Pull request merged**